File: Iniciante/1071.js
let li = Number(lines.shift());
let ls = Number(lines.shift());
let soma = 0;

if (ls > li) {
  for (let i = li+1; i < ls; i++) {
    if (i%2 != 0) {
    soma += i;
    }
  }
}

if (li > ls) {
  for (let i = ls+1; i < li; i++) {
    if (i%2 != 0) {
    soma += i;
    }
  }
}

console.log(soma);

File: Iniciante/1071.py
li = int(input())
ls = int(input())
soma = 0

if ls > li:
  for i in range(li+1, ls):
    if i%2 != 0:
      soma += i

if li > ls:
  for i in range(ls+1, li):
    if i%2 != 0:
      soma += i

print(soma)
